modules/mozillians/main.tf

```diff
@@ -1,7 +1,8 @@
 variable "environment" {}
 variable "vpc_id" {}
 variable "service_security_group_id" {}
-variable "elasticache_instance_size" {}
+variable "elasticache_redis_instance_size" {}
+variable "elasticache_memcached_instance_size" {}
```

Comment:
Shall we use the naming schema of the other projects and add `mozillians` in front ? (`mozillians_elasticache_memcached_instance_size`)